lib/src/lints/flutter/prefer_dedicated_media_query_method.dart

```diff
@@ -97,6 +119,30 @@ class _ReplaceWithDedicatedMethod extends DartFix {
     AnalysisError analysisError,
     List<AnalysisError> others,
   ) {
+    context.registry.addPrefixedIdentifier((node) {
+      if (!analysisError.sourceRange.intersects(node.sourceRange)) return;
+
+      final typeName =
+          node.prefix.staticType?.getDisplayString(withNullability: false);
+      if (typeName != 'MediaQueryData') return;
+
+      final methodName = node.identifier.name;
+
+      final expected = 'MediaQuery.${methodName}Of(context)';
```

Comment:
I am not quite sure if `expected` is a proper name here, perhaps we could use `dedicatedMethod` instead?

Reply:
we can make it dedicatedMethod.